packages/valory/services/trader_pearl/service.yaml

```diff
@@ -92,10 +99,10 @@ models:
         "token_address":"0x1b5DeaD7309b56ca7663b3301A503e077Be18cba", "order_address":"0x72201948087aE83f8Eac22cf7A9f2139e4cFA829",
         "nft_amount":"100", "payment_token":"0x0000000000000000000000000000000000000000",
         "price":"1000000000000000000"}}
```

Comment:
@0xArdi to verify if these are correct. Currently used in mainstream Pearl is on https://github.com/valory-xyz/trader/pull/344

Reply:
The values are not the same, e.g.:
https://github.com/valory-xyz/trader/blob/d52407875887eb07fc730c4c0747b65655d4ccc0/packages/valory/services/trader_pearl/service.yaml#L94

I guess the latest ones are the correct ones. In any case, it does not matter because they can be overridden. Moreover, the marketplace is not used, unless you plan to enable it on Pearl.
https://github.com/valory-xyz/trader/blob/d9cfdbd57ff443ffd46c16d24258432a4f207aac/packages/valory/services/trader_pearl/service.yaml#L104

Reply:
Further questions for Trader team about variables:
1. Is `blacklisting_duration` used?
2. Is `use_mech_marketplace`? Seems used in mech interact and agent yaml, but not referenced in service yaml.
3. mech_request_price has been set to 'null' in pearl trader (it was set to 0 before). Is this OK?